packages/compass-saved-aggregations-queries/src/index.spec.ts

```diff
@@ -1,3 +1,7 @@
+// this is to avoid BigInteger is not defined error, which is thrown in ipv6/lib/node/bigint.js
+// import chain: compass-query-history -> mongodb-data-service -> ssh-tunnel -> socksv5 -> ipv6 (which is pretty old and deprecated).
+globalThis.BigInteger = undefined;
```

Comment:
I feel like some sort of setup file would be a better place for this instead of just one of the test files